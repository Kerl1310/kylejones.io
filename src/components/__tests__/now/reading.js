import React from 'react';
import renderer from 'react-test-renderer';

import Reading from '../../now/reading';

const testBook = {
    isbn: '9780553803709',
    titleWithoutSeries: 'I, Robot',
    link: 'https://www.goodreads.com/book/show/41804.I_Robot'
}

const testAuthors = [{
    id: '16667',
    link: 'https://www.goodreads.com/author/show/16667.Isaac_Asimov',
    name: 'Isaac Asimov'
}]

describe('Reading', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Reading book={testBook} authors={testAuthors}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
