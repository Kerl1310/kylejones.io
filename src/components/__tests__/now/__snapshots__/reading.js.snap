// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reading renders correctly 1`] = `
<li>
  <strong>
    Reading
    : 
  </strong>
  <a
    className="book-link-9780553803709"
    href="https://www.goodreads.com/book/show/41804.I_Robot"
    rel="noopener noreferrer"
    target="_blank"
  >
    I, Robot
  </a>
   
  by
   
  <a
    className="author-link"
    href="https://www.goodreads.com/author/show/16667.Isaac_Asimov"
    id="author-link-16667"
    rel="noopener noreferrer"
    target="_blank"
  >
    Isaac Asimov
  </a>
</li>
`;
